Sources/MapboxNavigation/Expression.swift

```diff
@@ -30,4 +36,86 @@ extension Expression {
             }
         }
     }
+    
+    func localized(into locale: Locale?) -> Expression {
+        switch elements.first {
+        case .op(.literal):
+            return self
+        case .op(.get):
+            guard elements.count == 2,
+                  case let .argument(argument) = elements[1],
+                  case let .string(propertyName) = argument else {
+                break
+            }
+            
+            if propertyName == "name" || propertyName.starts(with: "name_") {
+                var localizedPropertyName = "name"
+                if locale?.identifier != "mul" {
+                    let preferences: [String]
+                    if let identifier = locale?.identifier {
+                        preferences = [identifier]
+                    } else {
+                        preferences = Locale.preferredLanguages
+                    }
```

Comment:
I'd just set default value for `preferences` instead of introducing else statement.

Reply:
It’s a matter of style, but in general, I prefer to minimize usage of local variables as much as possible to avoid mutability surprises further down in the code, even though it’s slightly more verbose.